actin/operational/share_report_with_nki

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+PROJECT="actin-nki"
+SOURCE_BUCKET="gs://${PROJECT}-analysis-pipeline-output"
+TARGET_BUCKET="gs://${PROJECT}-shared-data"
+LOOKUP_CMD="$(dirname "$0")/lookup_hash_for_actin_id"
+
+[[ $# -ne 1 ]] && echo "Provide the ACTIN id for the report" >&2 && exit 1
+
+report="${SOURCE_BUCKET}/report/${1}.actin.pdf"
+extended="${SOURCE_BUCKET}/extended/${1}.actin.extended.pdf"
+
+hash="$($LOOKUP_CMD "$1")"
+for source in "$report" "$extended"; do
+  echo $source
```

Comment:
minor: was this for debugging? in most of your echo's you add descriptive text.

Reply:
Thank-you.